experiments/AtmosLES/rising_bubble_theta_formulation.jl

```diff
@@ -0,0 +1,372 @@
+# [AtmosModel](@ref AtmosModel-docs) machinery to solve the fluid
+# dynamics of a thermal perturbation in a neutrally stratified background state
+# defined by its uniform potential temperature. We solve a flow in a box configuration -
+# this is representative of a large-eddy simulation. Several versions of the problem
+# setup may be found in literature, but the general idea is to examine the
+# vertical ascent of a thermal bubble (we can interpret these as simple
+# representation of convective updrafts).
+#
+# ## Description of experiment
+# 1) Dry Rising Bubble (circular potential temperature perturbation)
+# 2) Boundaries
+#    Top and Bottom boundaries:
+#    - `Impenetrable(FreeSlip())` - Top and bottom: no momentum flux, no mass flux through
+#      walls.
+#    - `Impermeable()` - non-porous walls, i.e. no diffusive fluxes through
+#       walls.
+#    Lateral boundaries
+#    - Laterally periodic
+# 3) Domain - 2500m (horizontal) x 2500m (horizontal) x 2500m (vertical)
+# 4) Resolution - 50m effective resolution
+# 5) Total simulation time - 1000s
+# 6) Mesh Aspect Ratio (Effective resolution) 1:1
+# 7) Overrides defaults for
+#    - CPU Initialisation
+#    - Time integrator
+#    - Sources
+#    - Smagorinsky Coefficient
+
+#md # !!! note
+#md #     This experiment setup assumes that you have installed the
+#md #     `ClimateMachine` according to the instructions on the landing page.
+#md #     We assume the users' familiarity with the conservative form of the
+#md #     equations of motion for a compressible fluid (see the
+#md #     [AtmosModel](@ref AtmosModel-docs) page).
+#md #
+#md #     The following topics are covered in this example
+#md #     - Package requirements
+#md #     - Defining a `model` subtype for the set of conservation equations
+#md #     - Defining the initial conditions
+#md #     - Applying source terms
+#md #     - Choosing a turbulence model
+#md #     - Adding tracers to the model
+#md #     - Choosing a time-integrator
+#md #     - Choosing diagnostics (output) configurations
+#md #
+#md #     The following topics are not covered in this example
+#md #     - Defining new boundary conditions
+#md #     - Defining new turbulence models
+#md #     - Building new time-integrators
+#md #     - Adding diagnostic variables (beyond a standard pre-defined list of
+#md #       variables)
+
+# ## [Loading code](@id Loading-code-rtb)
+
+# Before setting up our experiment, we recognize that we need to import some
+# pre-defined functions from other packages. Julia allows us to use existing
+# modules (variable workspaces), or write our own to do so.  Complete
+# documentation for the Julia module system can be found
+# [here](https://docs.julialang.org/en/v1/manual/modules/#).
+
+# We need to use the `ClimateMachine` module! This imports all functions
+# specific to atmospheric and ocean flow modeling.
+
+using ClimateMachine
+ClimateMachine.init(; parse_clargs = true)
```

Comment:
@kpamnany should this be changed? Or is it okay?

Reply:
This is fine I think. Only stuff in the `test/runtests.jl` hierarchy shouldn't parse command line arguments.